tests/ui/js/base-page-storyEditTests.js

```diff
@@ -863,6 +879,303 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
         return fluid.get(component, "story.model.content.0." + fieldName);
     };
 
+    var isEmptyBlockTestCases = [
```

Comment:
Should be attached to the public namespace e.g. `sjrk.storyTelling.base.page.storyEditTester.isEmptyBlockTestCases`.